source/js/buyers-guide/components/creep-vote/creep-vote.scss

```diff
@@ -70,14 +70,4 @@ $btn-shadow-width: 3px;
   .help-text {
     font-style: italic;
   }
-
```

Comment:
This change should not be here. Please make sure that the branch you're using for the PR just has the changes to the documentation.